homework/shelkovich_ilya/wrapperClasses/src/main/java/com/tutrit/java/quickstart/Application.java

```diff
@@ -0,0 +1,76 @@
+package com.tutrit.java.quickstart;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Application {
+
+    static Logger log = LoggerFactory.getLogger(Application.class);
+
+    public static void main(String[] args) {
+        Integer intByLiteral = 100;
+        Integer.valueOf(4);
+        Integer.parseInt("4");
+
+
+        // Не понял как продемонстрировать разницу присвоения по ссылке и методом valueOf
```

Comment:
All of this should be divided into tests

Reply:
improved it